src/main/java/com/jagrosh/vortex/automod/AutoMod.java

```diff
@@ -480,9 +483,11 @@ else if(offenses>settings.dupeDeleteThresh)
                         redirects = urlResolver.findRedirects(link);
                         for(String resolved: redirects)
                         {
+                            //TODO: add && preventInvites like for other check? (topic)
                             if(settings.inviteStrikes>0 && resolved.matches(INVITE_LINK))
                             {
-                                if(inviteResolver.resolve(message.getJDA(), resolved.replaceAll(INVITE_LINK, "$1")) != message.getGuild().getIdLong())
+                                long invite = inviteResolver.resolve(message.getJDA(), resolved.replaceAll(INVITE_LINK, "$1"));
```

Comment:
See TODO entry above... is it intentional that invite automod can't be disabled via topic, if invites are detected via resolved redirects?

Reply:
Moved the inviteStrikes > 0 checks into the preventInvites boolean as discussed in DMs